src/convertToObject.js

```diff
@@ -11,8 +11,30 @@
  *
  * @return {object}
  */
+/* eslint-enable */
```

Comment:
```suggestion
```

Reply:
Do You have some errors with this text?

Reply:
No, it's only for comments. Linter has shown an error.